index.js

```diff
@@ -59,10 +59,16 @@ const retryStatusCodes = new Set([
 	503,
 	504
 ]);
+const retryAfterStatusCodes = new Set([
+	413,
+	429,
+	503
+]);
 
 class HTTPError extends Error {
 	constructor(response) {
 		super(response.statusText);
+		Object.setPrototypeOf(this, HTTPError.prototype);
```

Comment:
This line is useless

Reply:
@szmarczak fixed. I forgot to remove it.